src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -375,6 +436,14 @@ AutoDownloadRule AutoDownloadRule::fromJsonObject(const QJsonObject &jsonObj, co
         feedURLs << urlVal.toString();
     rule.setFeedURLs(feedURLs);
 
+    const QJsonValue previouslyMatchedVal = jsonObj.value(Str_PreviouslyMatched);
+    QStringList previouslyMatched;
+    if (previouslyMatchedVal.isString())
+        previouslyMatched << previouslyMatchedVal.toString();
+    else foreach (const QJsonValue &val, previouslyMatchedVal.toArray())
```

Comment:
Then **if** branch must be enclosed with curly braces too.

P.S. Personally I like curent form, but if it doesn't follow qBittorrent coding style it should be fixed.